lib/puppet/gettext/module_translations.rb

```diff
@@ -0,0 +1,30 @@
+require 'puppet/gettext/config'
+
+module Puppet::ModuleTranslations
+  def self.from_modulepath(modules)
+    modules.each do |mod|
+      return unless mod.forge_name
+
+      module_name = mod.forge_name.gsub('/', '-')
```

Comment:
Can this be simplified to `mod.name`? IIRC the [`forge_name`](https://github.com/puppetlabs/puppet/blob/master/lib/puppet/module.rb#L230) is derived from the `name` with `-` replaced with '/', which the above line is undoing.

Reply:
No, `mod.name` doesn't contain the forge namespace, which we include in our translation file names to avoid collisions.